### PR TITLE
MINOR: Remove redundant code from BrokerApiVersionsCommand

### PR DESCRIPTION
**Pull request by wenbingshen**
Remove some redundant parameters and configurations

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by wenbingshen** [comment]
Dear @ijuma @chia7712 Can you take a look at it？Thanks.

File: core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala [hunk 1]
```diff
@@ -17,32 +17,27 @@
 
 package kafka.admin
 
-import java.io.PrintStream
```
Comment (chia7712) [inline core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala]: I'd like this order optimization. However, it is not related to this PR and it may produce conflicting files to other PRs. Hence, could you please revert this change?

**Review by chia7712 — COMMENTED** [review]
@wenbingshen thanks for your patch. one small comment is left.

File: core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala [hunk 2]
```diff
@@ -17,32 +17,27 @@
...
-import org.apache.kafka.common.utils.LogContext
-import org.apache.kafka.common.utils.{KafkaThread, Time}
-import org.apache.kafka.common.Node
-import org.apache.kafka.common.message.ApiVersionsResponseData.ApiVersionCollection
-import org.apache.kafka.common.requests.{AbstractRequest, AbstractResponse, ApiVersionsRequest, ApiVersionsResponse, MetadataRequest, MetadataResponse}
+import org.apache.kafka.common.protocol.Errors
+import org.apache.kafka.common.requests._
+import org.apache.kafka.common.utils.{KafkaThread, LogContext, Time, Utils}
 
+import java.io.{IOException, PrintStream}
```
Comment (wenbingshen) [inline core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala]: > I'd like this order optimization. However, it is not related to this PR and it may produce conflicting files to other PRs. Hence, could you please revert this change?

@chia7712 Thank you so much for your comment. Now both IOException and PrintStream are inside the same {}. 
Please help me to review this PR again. :)

**Review by wenbingshen — COMMENTED** [review]
@chia7712 Thank you for helping me review this pr. :)

File: core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala (hunk 2 above)
Comment (chia7712) [reply core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala]: Sorry for my unclear comment. My point was that this PR re-order all imports of this class. It would be better to revert all changes related to re-order.

Comment (wenbingshen) [reply core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala]: Sorry, blame me, I understand what you mean now, thank you for your feedback, I will make changes and submit it later today. :)

**Comment by wenbingshen** [comment]
Dear @chia7712 According to your instructions, I have reverted all imports. In your free time, please help me to review again. Thanks. :)

**Comment by wenbingshen** [comment]
> Dear @chia7712 According to your instructions, I have reverted all imports. In your free time, please help me to review again. Thanks. :)

**Comment by wenbingshen** [comment]
> > Dear @chia7712 According to your instructions, I have reverted all imports. In your free time, please help me to review again. Thanks. :)

**Comment by wenbingshen** [comment]
@chia7712  These test failures seem to have nothing to do with this pr. Apart from merging the backbone, can we still trigger the inspection? I do not know what to do.

**Comment by chia7712** [comment]
> These test failures seem to have nothing to do with this pr. Apart from merging the backbone, can we still trigger the inspection? I do not know what to do.

It seems to me the failure is unrelated to this PR. Thanks for your contribution. merge to trunk
